modules/network-observability-lokistack-create.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-lokistack-create_{context}"]
+
+= Create a LokiStack custom resource 
+
+.Procedure
+
+. Navigate to *Operators* -> *Installed Operators*. In the Details, under *Provided APIs*, select `LokiStack` and click *Create LokiStack*. 
+. Ensure the following fields are specified in either *Form View* or *YAML view*:
++
+[source,yaml]
+----
+  apiVersion: loki.grafana.com/v1
+  kind: LokiStack
+  metadata:
+    name: loki
```

Comment:
hmm why would we want to do this distinction? For me it's fine to say LokiStack ~= Loki, as this is what creates the actual loki workloads. Also - just to be pragmatic - that would require to review again the whole PR to make sure everything is changed accordingly, maybe something to avoid given the short time left

Reply:
Sure, no worries.

Reply:
I will make note to address naming conventions post GA just to make sure things are ok. But if its okay as is that's good for now, I agree to leave it.